### DISCUSSION TITLE
Add expiration in Discussions

### DISCUSSION THREAD

**Discussion by ghost** [community/community#16739] [Discussions]
Hi all!

#### 1. Feature name
Discussions with expiration

#### 2. Feature description
I would like to create temporary discussions, after a while they are deleted.

#### 3. Concept
There are several software that have this feature, we can mention Pastebin, where you post something that is later deleted over time

#### 3.1 ui/ux before
![image](https://user-images.githubusercontent.com/95204402/168693106-9628f99d-173b-46e0-93c0-6eaebb35cb76.png)

#### 3.2 ui/ux after
![image](https://user-images.githubusercontent.com/95204402/168693083-680760b8-59cf-4261-afbe-577ecb9515e1.png)

#### 4. Notes
1. I would like to help GitHub in a positive way
2. I'm not promoting any software or company here, I just cited Pastebin as a bibliographic reference
3. If the topic is in the closing phase, there are 3 options:
    - option 1: prevent new replies to the topic - which usually happens in any internet forum.
    - option 2: remove the topic automatically according to a set time.
    - option 3: leave the topic archived - so this topic can be updated after a while.
4. Option 3 is already being requested, option 1 has already been considered, reference [option1](https://github.com/github/feedback/discussions/3097). What I'm doing here is requesting option 2, reference [option2](https://github.com/github/feedback/discussions/16739)!

#### 5. Reference
- https://pastebin.com/

**Comment by dianaledist** [comment]
Interesting proposal!! Maybe it could be a dropdown menu :)

**Reply by ghost** [reply]
@dianaledist thank you for feedback ;D I'm happy with your comment

**Comment by SleazeStiKs** [comment]
Very cice idea! Temporary discussions could help keeping conversations on Github without the need to use other apps to communicate on small scale problems. I'm sure this would speed up the development process.

**Reply by ghost** [reply]
@SleazeStiKs thanks for the comment, I'm happy with your comment too, you are amazing ;D

**Comment by ghost** [comment]
Great idea!

**Comment by ghost** [comment]
Hey everyone! I find this: https://github.com/community/community/discussions/3867 and it looks similar to the one requested here.
